website/content/gateway/api_reference/changes/changes.md

```diff
@@ -10,6 +10,10 @@ Follow coming changes on the [source code repository](https://github.com/clearha
 
 Sorted by descending timestamp.
 
+#### Addition of Mastercard debits
+Mastercard debits support added. The parameter `recipient[account_number_type]`
```

Comment:
Date to be added before merge.

Reply:
Yep. As a side note I noticed that allowing voids on authorizations does not have a date.